assets/js/payment-methods-demo/payment-methods/index.js

```diff
@@ -4,22 +4,28 @@
 import { paypalSvg } from './paypal';
 import { ccSvg } from './cc';
 
-export const PaypalPaymentMethod = {
-	tab: () => <img src={ paypalSvg } alt="" />,
-	content: () => (
+export const paypalPaymentMethod = {
+	id: 'paypal',
+	label: () => <img src={ paypalSvg } alt="" />,
```

Comment:
Same questions here about `label` and `stepContent`, capitalization, and react components versus nodes.